default/AI/ResourcesAI.py

```diff
@@ -55,8 +55,8 @@ def __init__(self, planet):
 
     def can_change_focus(self):
         """Can the planet change focus?  Is it's population non zero?"""
-        #TODO Could this be changed to current population instead of target population?
-        #It uses the target population to avoid situations where an unsustainable population is dying?
+        # It uses the target population versus current population to avoid situations
+        # where an unsustainable population is dying?
```

Comment:
It's my recollection that a while in the past, when a colony died down to an outpost it still retained its last-set focus, which could not be changed, hence a test like the one here (though I don't recall why it would look at target pop rather than current pop).  Nowadays the focus is reset to nothing in that case, so I think this test would no longer be needed (or perhaps simply be changed to look for nonempty AvailableFoci).